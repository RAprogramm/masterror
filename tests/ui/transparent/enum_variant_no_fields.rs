use masterror::Error;

#[derive(Debug, Error)]
enum TransparentEnumUnit {
    #[error(transparent)]
    Variant
}
