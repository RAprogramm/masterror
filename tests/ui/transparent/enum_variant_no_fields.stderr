error: using #[error(transparent)] in variant `Variant` requires exactly one field
 --> tests/ui/transparent/enum_variant_no_fields.rs:5:5
  |
5 |     #[error(transparent)]
  |     ^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/transparent/enum_variant_no_fields.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/transparent/enum_variant_no_fields.rs`
